pontoon/base/views.py

```diff
@@ -988,50 +994,119 @@ def transvision(request):
         return HttpResponse("error")
 
 
+def download_file(prefixes, dirnames, relative_path):
+    for prefix in prefixes:
+        for dirname in dirnames:
+            url = os.path.join(prefix.replace('{locale_code}', dirname), relative_path)
```

Comment:
- Don't use `os.path.join` for URLs. `'/'.join` is more appropriate.
- Why replace instead of `string.format`?